app/models/draft-registration.ts

```diff
@@ -23,6 +35,21 @@ export default class DraftRegistrationModel extends OsfModel {
 
     @belongsTo('registration-schema', { inverse: null })
     registrationSchema!: DS.PromiseObject<RegistrationSchemaModel> & RegistrationSchemaModel;
+
+    @hasMany('contributor', { inverse: 'node' })
+    contributors!: DS.PromiseManyArray<ContributorModel>;
+
+    @hasMany('contributor', { inverse: null })
+    bibliographicContributors!: DS.PromiseManyArray<ContributorModel>;
+
+    @hasMany('institution', { inverse: 'nodes' })
```

Comment:
```suggestion
    @hasMany('institution', { inverse: null })
```
institutions don't point to drafts